src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -119,7 +119,6 @@ class Task extends React.Component<Props> {
                     onDelete={onDelete}
                     onEdit={onEdit}
                     permissions={taskPermissions}
-                    messageHeader={<TaskDueDate dueDate={due_at} status={status} />}
```

Comment:
Can the `messageHeader` prop be removed from `Comment`? I think we added it recently to support the new task design?

Reply:
I can take it out, but was worried that since it was both requested to add this *and* remove it, removing it entirely might be premature